### DISCUSSION TITLE
Edge-config get vs getAll

### DISCUSSION THREAD

**Discussion by willemliufdmg** [vercel/next.js#49698] [Help]
### Summary

Hi, I can't find it in the documentation but the Vercel pricing states that costs are based on the number of Edge Config reads.

Does each call to the `get` function of the EdgeConfigClient (`@vercel/edge-config`) count as 1 read? And does each subsequent call to `get` count as an extra read as well? Or does the SDK do some caching and are all calls made within the same request in the same file (e.g. Middleware) count as 1?

Assuming that each `get` does count as a separate read. If I do a `getAll` and pass along an array with `keys`. Does this count as a single read? Or does each `key` count as a read?

**Comment by wottpal** [comment]
Did you ever find an answer?

**Reply by willemliufdmg** [reply]
I never got an answer. But I did find out that `getAll` counts as a single request, regardless how many keys you pass to it.

I haven't tested if multiple `get`/`getAll` within the same user-request counts as a single request or not.

**Reply by wottpal** [reply]
Ok, great, that's good news for us already!
